easybuild/easyconfigs/g/GCCcore/GCCcore-11.3.0.eb

```
@@ -0,0 +1,64 @@
+easyblock = 'EB_GCC'
+
+name = 'GCCcore'
+version = '11.3.0'
+
+homepage = 'https://gcc.gnu.org/'
+description = """The GNU Compiler Collection includes front ends for C, C++, Objective-C, Fortran, Java, and Ada,
+ as well as libraries for these languages (libstdc++, libgcj,...)."""
+
+toolchain = SYSTEM
+
+source_urls = [
+    'https://ftpmirror.gnu.org/gnu/gcc/gcc-%(version)s',  # GCC auto-resolving HTTP mirror
+    'https://ftpmirror.gnu.org/gnu/gmp',  # idem for GMP
+    'https://ftpmirror.gnu.org/gnu/mpfr',  # idem for MPFR
+    'https://ftpmirror.gnu.org/gnu/mpc',  # idem for MPC
+    'ftp://gcc.gnu.org/pub/gcc/infrastructure/',  # GCC dependencies
+    'http://gcc.cybermirror.org/infrastructure/',  # HTTP mirror for GCC dependencies
```

This should be an alternative `https://` URL, this `http://` one doesn't seem to work anymore (and we should fix that in the existing `GCCcore` easyconfigs too):

```suggestion
    'https://gcc.gnu.org/pub/gcc/infrastructure/',  # HTTP mirror for GCC dependencies
```

That's important if you don't have `isl-0.24.tar.gz` downloaded already...

---

Done in https://github.com/easybuilders/easybuild-easyconfigs/pull/15311/commits/a2d5095030c2cc58d9111e85395ba90a64fcc377

---

Update the URL in the existing `GCCcore` easyconfigs done in https://github.com/easybuilders/easybuild-easyconfigs/pull/15320